$OpenBSD: patch-src_libpiano_crypt_c,v 1.7 2013/06/18 13:17:45 dcoppa Exp $

Zap useless include
(upstream git commit a1e73fc8d72a5253cae29c9bab5b260ad297c5e2)

--- src/libpiano/crypt.c.orig	Sun May 19 12:58:18 2013
+++ src/libpiano/crypt.c	Tue Jun 18 14:58:00 2013
@@ -23,7 +23,6 @@ THE SOFTWARE.
 
 #include <string.h>
 #include <assert.h>
-#include <gcrypt.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <stdint.h>
